src/css/controls/states.less

```diff
@@ -126,10 +126,8 @@
     }
 
     .jw-controls {
```

Comment:
Do you need to set this gain? Looks like it's handled in https://github.com/jwplayer/jwplayer/pull/2404/files#diff-ed218305fc3cedaf045e81cf450e0087R166

Reply:
@johnBartos one is for state complete, the other is state error. we could probably group them but i know @radium-v  will be changing the background effect on idle/complete states so i preferred to defer that tech debt for now